apps/sms/index.html

```diff
@@ -128,7 +129,7 @@
 
   <body role="application" class="skin-comms theme-communications">
     <article id="main-wrapper" class="wrapper" data-position='right'>
-      <section id="thread-list" class="panel default-panel" role="region">
+      <section class="panel panel-thread-list" role="region">
```

Comment:
> Yeah, I thought of it, but I thought it could be for a separate patch, because we need to change every place where isCurrentPanel and toPanel are used.

I'm fine with this as well, could you please file papercuts bug then?